program/program.wake

```diff
@@ -39,6 +39,7 @@ tuple TestProgramPlan =
   global Entry:       Option (Pair String Integer)
   global Rom:         Option String
   global Ram:         Option String
+  global StdoutPath:  Option String
```

Comment:
Could we give this a slightly different name? If I'm reading the code correctly, it looks like this is meant to be the node name in DTS. However, when I just read the name of this property, this sounds like it would be the filepath to which you would output the stdout of the process. In either case, let's at the very least document what this means.